packages/client/ui/react-ui/src/providers/CrossmintAuthProvider.tsx

```diff
@@ -17,10 +17,13 @@ export type CrossmintAuthWalletConfig = {
     showWalletModals?: boolean;
 };
 
+export type LoginMethod = "email" | "google" | "farcaster";
```

Comment:
As SDKs are used in crossbit-main, how about having the type centralized in one place here in the sdks and the fetching that in crossbit-main?

Reply:
Good call, I can add it my main concern right now is if we wanted to add a auth method quickly from main repo we'd be bottlenecked by having to add the type in SDK. I'll make a ticket to follow this up later 👍